docs/README.md

```diff
@@ -2,9 +2,10 @@
 
 This is the public documentation for **Expo**, its SDK, client, and services (**EAS**). This documentation is built using Next.js and you can access it online at https://docs.expo.dev/.
 
-> **Note** **Contributors:** Please make sure that you edit the docs in the **pages/versions/unversioned** for SDK reference if you want your changes to apply to the next SDK version too!
+> [!NOTE] 
+> **Contributors:** Please make sure that you edit the docs in the **pages/versions/unversioned** for SDK reference if you want your changes to apply to the next SDK version too!
 
-> **Note**
+> [!NOTE]
```

Comment:
```suggestion
> [!TIP]
```

Let's use an another callout theme to avoid stacking two same types below each other, other options:
* https://github.com/orgs/community/discussions/16925